### ISSUE TITLE
Incorrect gamma when resizing?

### ISSUE THREAD

**Issue by chakflying** [android/renderscript-intrinsics-replacement-toolkit#13]
Using test image from [this website](http://www.ericbrasseur.org/gamma.html), Resize seems to use linear gamma instead of sRGB gamma.

| original | resized |
| ------- | ------- |
| ![gamma_dalai_lama_gray](https://user-images.githubusercontent.com/3271800/147762365-06d86596-40ac-4b66-8174-991b94fe3378.jpg) | ![test-resize53078175886776102](https://user-images.githubusercontent.com/3271800/147762389-4e93772d-20f1-4be6-aa7c-a673d01d0c6f.png) |

**Comment by jeanlucbr** [comment]
RenderScript uses bicubic interpolation when resizing.

Do you have a small test code you can provide that replicates the problem?

**Comment by ZXHHYJ** [comment]
这个方法似乎有bug，返回的bitmap一直都是灰色的，不知道现在修复了没有

**Comment by Kottsone** [comment]
Yes, as mentioned in the documentation, the implementation is a simple bicubic interpolation of the RGB value. As a replacement toolkit, we would like the behavior to match with the original RenderScript resize intrinsics. We could have another API to resize an image while preserving the gamma. Contributions are welcome.
